src/time.cr

```diff
@@ -252,8 +252,13 @@ struct Time
   # :nodoc:
   DAYS_PER_4_YEARS = 365*4 + 1
 
-  # :nodoc:
-  UNIX_SECONDS = SECONDS_PER_DAY.to_i64 * (1969*365 + 1969/4 - 1969/100 + 1969/400)
+  # This constant is defined to be "1970-01-01 00:00:00 UTC" on all systems with respect to the system clock.
```

Comment:
What is `on all systems with respect to the system clock` supposed to mean? The constant always represents the date `1970-01-01`. That doesn't depend on the system clock.

Reply:
I've taken this from the Rust's documentation https://doc.rust-lang.org/std/time/constant.UNIX_EPOCH.html.

Reply:
Yeah, but what does it mean for the Crystal implementation? I can't make any sense of that.
In Rust it is an instance of `SystemTime` which is inherently system-dependent. That's probably the reason why they wrote that.

Reply:
Please remove the "on all systems with respect to the system clock" wording.